--- ./rcdb/java/src/org/rcdb/SQLiteProvider.kt	2024-04-19 18:41:37.249805000 -0400
+++ ./rcdb/java/src/org/rcdb/SQLiteProvider.kt	2024-04-19 18:38:15.638339000 -0400
@@ -32,19 +32,19 @@
 
         prsConditionType = con.prepareStatement("SELECT id, name, value_type FROM condition_types")
         prsCondition = con.prepareStatement("SELECT id, bool_value, float_value, int_value, text_value, time_value FROM conditions WHERE run_number = ? AND condition_type_id = ?")
-        prsFileNames = con.prepareStatement("SELECT files.path AS files_path "
-                                                + "FROM files, files_have_runs AS files_have_runs_1 "
-                                                + "WHERE files.id = files_have_runs_1.files_id "
-                                                + "AND ? = files_have_runs_1.run_number "
-                                                + "ORDER BY files.id DESC")
-        prsFile = con.prepareStatement("SELECT files.id AS files_id, "
-                                           + "       files.path AS files_path, "
-                                           + "       files.sha256 AS files_sha256, "
-                                           + "       files.content AS files_content "
-                                           + "FROM files, files_have_runs AS files_have_runs_1 "
-                                           + "WHERE files.path = ? AND files.id = files_have_runs_1.files_id "
-                                           + "      AND ? = files_have_runs_1.run_number "
-                                           + "ORDER BY files.id DESC")
+//        prsFileNames = con.prepareStatement("SELECT files.path AS files_path "
+//                                                + "FROM files, files_have_runs AS files_have_runs_1 "
+//                                                + "WHERE files.id = files_have_runs_1.files_id "
+//                                                + "AND ? = files_have_runs_1.run_number "
+//                                                + "ORDER BY files.id DESC")
+//        prsFile = con.prepareStatement("SELECT files.id AS files_id, "
+//                                           + "       files.path AS files_path, "
+//                                           + "       files.sha256 AS files_sha256, "
+//                                           + "       files.content AS files_content "
+//                                           + "FROM files, files_have_runs AS files_have_runs_1 "
+//                                           + "WHERE files.path = ? AND files.id = files_have_runs_1.files_id "
+//                                           + "      AND ? = files_have_runs_1.run_number "
+//                                           + "ORDER BY files.id DESC")
         postConnect()
     }
 }
